### PR TITLE
OBSDOCS-1281: Logging 5.8.12 release notes

### PR DESCRIPTION
**Pull request by shreyasiddhartha**
Change type: Doc update; Logging Z-Stream Release Notes - 5.8.12
Doc JIRA: https://issues.redhat.com/browse/OBSDOCS-1281

Fix Version: 4.12, 4.13, 4.14, 4.15

Doc Preview: https://81482--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/logging/logging_release_notes/logging-5-8-release-notes.html#cluster-logging-release-notes-5-8-12_logging-5-8-release-notes

SME Review: @periklis 
QE Review: @anpingli 
Peer Review: @ShaunaDiaz

### CODE REVIEW

**Review by periklis — COMMENTED** [review]
LGTM

File: modules/logging-release-notes-5-8-12.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
+// module included in logging-5-8-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="cluster-logging-release-notes-5-8-12_{context}"]
+= Logging 5.8.12
+This release includes a link:https://access.redhat.com/errata/RHBA-2024:6459[OpenShift Logging Bug Fix Release 5.8.12] and link:https://access.redhat.com/errata/RHBA-2024:6094[OpenShift Logging Bug Fix Release 5.8.12].
+
+[id="openshift-logging-5-8-12-bug-fixes_{context}"]
+== Bug fixes
+
+* Before this update, the collector used internal buffering with a `drop_newest` setting to reduce high memory usage, which led to significant log loss. With this update, the collector returns to its default behavior to prevent this issue. (link:https://issues.redhat.com/browse/LOG-6026[LOG-6026])
```
Comment (ShaunaDiaz) [inline modules/logging-release-notes-5-8-12.adoc:10]: What was the default behavior and why did it change to internal buffering?

File: modules/logging-release-notes-5-8-12.adoc [hunk 2]
```diff
@@ -0,0 +1,63 @@
+// module included in logging-5-8-release-notes.adoc
+:_mod-docs-content-type: REFERENCE
+[id="cluster-logging-release-notes-5-8-12_{context}"]
+= Logging 5.8.12
+This release includes a link:https://access.redhat.com/errata/RHBA-2024:6459[OpenShift Logging Bug Fix Release 5.8.12] and link:https://access.redhat.com/errata/RHBA-2024:6094[OpenShift Logging Bug Fix Release 5.8.12].
```
[the remark this answers is not in the document]
Comment (shreyasiddhartha) [reply modules/logging-release-notes-5-8-12.adoc:5]: Removing the article altogether, if that's okay.

File: modules/logging-release-notes-5-8-12.adoc (hunk 1 above)
Comment (anpingli) [reply modules/logging-release-notes-5-8-12.adoc:10]: default behavior: buffer is customized. it is what is designed for the output plugin in upstream.

**Pull request merged**